luigi/execution_summary.py

```diff
@@ -89,34 +91,39 @@ def _get_str(task_dict, extra_indent):
     """
     This returns a string for each status
     """
+    summary_length = int(configuration.get_config().get('core', 'summary-length', 5))
```

Comment:
Is it?  I can only find this in use by HDFS.  Every other instance I can find of configuration does it in a similar fashion.

From [notifications.py](https://github.com/spotify/luigi/blob/master/luigi/notifications.py#L45):
        return configuration.get_config().get('core', 'email-type', 'plain')

From [postgres.py](https://github.com/spotify/luigi/blob/master/luigi/postgres.py#L107)
    marker_table = luigi.configuration.get_config().get('postgres', 'marker-table', 'table_updates')

[interface.py](https://github.com/spotify/luigi/blob/master/luigi/interface.py#L156)
    if not configuration.get_config().getboolean(
            'core', 'no_configure_logging', False):

That said, I should have used 'getint' rather than int(... get_config.get(...))  An update will be coming shortly.